wp-content/themes/pub/wporg-learn-2020/archive-course.php

```diff
@@ -20,10 +20,14 @@
 
 <main id="main" class="site-main">
 	<section>
-		<div class="section-heading section-heading--with-space">
-			<?php the_archive_title( '<h1 class="section-heading_title h2">', '</h1>' ); ?>
+		<div class="row align-middle between section-heading">
+			<div class="section-heading section-heading--with-space">
+				<?php the_archive_title( '<h1 class="section-heading_title h2">', '</h1>' ); ?>
+			</div>
+			<?php if ( is_user_logged_in() ) : ?>
+				<a class="section-heading_link" href="/my-courses/"><span aria-hidden="true"><?php esc_html_e( 'My Courses', 'wporg-learn' ); ?></span><span class="screen-reader-text"><?php esc_html_e( 'My Courses', 'wporg-learn' ); ?></span></a>	
```

Comment:
No need for screen reader specific text here. Only do this if there are icons involved or screen reader users truly need different wording.

```suggestion
				<a class="section-heading_link" href="/my-courses/"><span><?php esc_html_e( 'My Courses', 'wporg-learn' ); ?></span></a>
```

Reply:
Thanks for the feedback @alexstine I will implement your suggestions.